packages/react/src/components/Masthead/README.md

```diff
@@ -17,7 +17,7 @@ Here's a quick example to get you started.
 @import '@carbon/ibmdotcom-styles/scss/components/masthead/masthead.scss';
 ```
 
-> 💡 Only import fonts once per usage. Don't forget to import the Masthead
+> `💡 Only import fonts once per usage. Don't forget to import the Masthead`
```

Comment:
![Screen Shot 2020-04-28 at 9 27 10 AM](https://user-images.githubusercontent.com/1641214/80492753-77d46600-8932-11ea-950c-4cbf8fe3204a.png)

The change from this seems odd, is this necessary?